core/src/main/java/nl/nn/adapterframework/doc/model/FrankDocModel.java

```diff
@@ -183,27 +190,25 @@ public FrankElement findFrankElement(String fullName) {
 		return allElements.get(fullName);
 	}
 
-	List<FrankAttribute> createAttributes(Class<?> clazz, FrankElement attributeOwner) throws ReflectiveOperationException {
+	List<FrankAttribute> createAttributes(FrankClass clazz, FrankElement attributeOwner) throws DocletReflectiveOperationException {
```

Comment:
Waarom wordt ReflectiveOperationException hier een ReflectiveOperationException? Op dit punt zou het implementatie onafhankelijk moeten zijn

Reply:
Ik heb de naam van de exception veranderd in FrankDocException. Dit laat zien dat het onafhankelijk is van of je met reflection bezig bent of met een doclet.